cheroot/ssl/pyopenssl.py

```diff
@@ -150,14 +150,24 @@ def readline(self, size=-1):
             size,
         )
 
-    def sendall(self, *args, **kwargs):
-        """Send whole message to the socket."""
+    def read(self, *args, **kwargs):
+        """Read from the wrapped socket, with retry."""
```

Comment:
I think that retry is an unnecessary implementation detail here.

Reply:
ok